src/test/firecracker/virtio-block-parse/main.rs

```diff
@@ -1,13 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-// --unwinding-assertions
-//
+
+// rmc-flags: --no-overflow-checks
```

Comment:
Whenever we remove a check, lets have a comment explaining why, and maybe a tracking issue if its worth it